### ISSUE TITLE
Server side apply force and field manager

### ISSUE THREAD

**Issue by stevehipwell** [gavinbunney/terraform-provider-kubectl#139]
It is currently impossible to switch to using `server_side_apply = true` for resources created without it due to the change in field manager name (`Hasicorp` to `kubectl`) and there not being a `server_side_apply_force` option.

It would also be useful to allow the field manager name to be customised.

**Comment by lawliet89** [comment]
Something like the `force_conflict` option on `kubernetes_manifest` and `--force-conflict`on `kubectl` will be useful.

**Comment by smehboub** [comment]
Hello,

Thank you for your job to all and to @kahirokunn
@gavinbunney, is it possible to create a release with this: https://github.com/gavinbunney/terraform-provider-kubectl/pull/148

Thanks in advance.
Rgs.

**Comment by aidanmelen** [comment]
@gavinbunney could you create a release please? I am running into an error that is solved in #148 

```hcl
╷
│ Error: kube-system/aws-auth failed to run apply: Apply failed with 1 conflict: conflict with "vpcLambda" using v1: .data.mapRoles
│ Please review the fields above--they currently have other managers. Here
│ are the ways you can resolve this warning:
│ * If you intend to manage all of these fields, please re-run the apply
│   command with the `--force-conflicts` flag.
│ * If you do not intend to manage all of the fields, please edit your
│   manifest to remove references to the fields that should keep their
│   current managers.
│ * You may co-own fields by updating your manifest to match the existing
│   value; in this case, you'll become the manager if the other manager(s)
│   stop managing the field (remove it from their configuration).
│ See http://k8s.io/docs/reference/using-api/server-side-apply/#conflicts
```
